charts/kubernetes-agent/templates/auto-upgrader-rolebindings.yaml

```diff
@@ -2,6 +2,10 @@
 {{- $autoUpgraderServiceAccountName := include "kubernetes-agent.autoUpgraderServiceAccountName" . -}}
 {{- $autoUpgraderSelfRoleName := include "kubernetes-agent.autoUpgraderSelfRoleName" . -}}
 {{- $autoUpgraderTargetRoleName := include "kubernetes-agent.autoUpgraderTargetRoleName" . -}}
+{{- $targetNamespaces := .Values.scriptPods.serviceAccount.targetNamespaces -}}
+{{- if not $targetNamespaces }}
+{{- $targetNamespaces = .Values.global.targetNamespaces -}}
+{{- end }}
```

Comment:
Is this not the `"kubernetes-agent.targetNamespaces"` helper?

Reply:
Yep - as discussed, fixed!